src/Stratis.Bitcoin.Features.Dns/IFullNodeBuilderExtensions.cs

```diff
@@ -30,6 +31,7 @@ public static IFullNodeBuilder UseDns(this IFullNodeBuilder fullNodeBuilder)
                     services.AddSingleton<DnsSettings>();
                     services.AddSingleton<IUdpClient, DnsSeedUdpClient>();
                     services.AddSingleton<IWhitelistManager, WhitelistManager>();
+                    services.AddSingleton<IPeerBanning, PeerBanning>();
```

Comment:
You don't need to do this peer banning is available by default in base

Reply:
ok cool, I couldn't see it there